rllib/agents/ppo/tests/test_ppo.py

```diff
@@ -191,13 +207,21 @@ def test_ppo_exploration_setup(self):
 
     def test_ppo_free_log_std(self):
         """Tests the free log std option works."""
-        config = copy.deepcopy(ppo.DEFAULT_CONFIG)
-        config["num_workers"] = 0  # Run locally.
-        config["gamma"] = 0.99
-        config["model"]["fcnet_hiddens"] = [10]
-        config["model"]["fcnet_activation"] = "linear"
-        config["model"]["free_log_std"] = True
-        config["model"]["vf_share_layers"] = True
+        config = (
+            ppo.PPOConfig()
+            .rollouts(
+                num_rollout_workers=0,
+            )
+            .training(
+                gamma=0.99,
+                model=dict(
+                    fcnet_hiddens=[10],
+                    fcnet_activation="linear",
```

Comment:
we should keep the more complex, hence nested, cases in mind for a second round of simplifications. I.e. in this case a more intuitive `ModelConfig` would certainly be appreciated by our users.

Reply:
hmm, on second thought, maybe we can investigate this a little more rn. Apart from the models themselves, I'm not sure we have many more such complex cases.

Reply:
Absolutely! Didn't want to do everything in this PR. It's already quite large.

Reply:
* Exploration
* Input/output configs
* Env config
* Evaluation config

Reply:
makes perfect sense.